src/peer_manager.rs

```diff
@@ -440,38 +420,38 @@ impl PeerManager {
                                client_auth: Authority<XorName>)
                                -> Result<BTreeSet<PublicId>, RoutingError> {
         self.remove_unapproved_candidates(&candidate_name);
-        {
-            let target_size = resource_proof_target_size(self.routing_table.min_section_size(),
-                                                         self.routing_table.our_section().len());
-            let candidate = self.candidates
-                .entry(candidate_name)
-                .or_insert_with(|| Candidate::new(client_auth, target_size));
-            candidate.state = CandidateState::AcceptedAsCandidate;
-        }
+        self.candidates
+            .entry(candidate_name)
+            .or_insert_with(|| Candidate::new(client_auth))
+            .state = CandidateState::AcceptedAsCandidate;
         let our_section = self.routing_table.our_section();
         Ok(self.get_pub_ids(our_section))
     }
 
     /// Verifies proof of resource.  If the response is too close to the overall resource proof
     /// timeout, or if it's not the current candidate, or if it fails validation, returns `Err`.
-    /// Otherwise returns a tuple containing the candidate's `PublicId`, its client `Authority` and
-    /// the `PublicId`s of all routing table entries.
+    /// Otherwise returns the target size and difficulty.
```

Comment:
I think "If the response is too close to the overall resource proof timeout" is no longer true (was missed in my last commit).